extensions/jasypt/deployment/src/test/java/org/apache/camel/quarkus/component/jasypt/JasyptPasswordNotProvidedTest.java

```diff
@@ -25,11 +28,27 @@
 public class JasyptPasswordNotProvidedTest {
     @RegisterExtension
     static final QuarkusUnitTest CONFIG = new QuarkusUnitTest()
+            .overrideConfigKey("greeting.secret", "ENC(GKJfy64eBDzxUuQCfArd6OjnAaW/oM9e)")
             .setExpectedException(IllegalStateException.class)
-            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class));
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class).addClass(JasyptRoutes.class));
 
     @Test
     void passwordNotProvidedThrowsException() {
         // Nothing to test as we just verify the application fails to start
     }
+
+    public static final class JasyptRoutes extends RouteBuilder {
```

Comment:
Just curious why we add `RouteBuilder` classes to tests that do nothing other that assert the application fails to start correctly?

Can we remove them from here and the other tests they got added to?

Reply:
Because the secret key is now lazy, it does not initialize when the Config starts (on application start), but when it is called. Without the route, the application starts fine (instead of the expected exception) because the secret handle is never invoked, so we need to force it.